server/index.js

```diff
@@ -1,9 +1,12 @@
 import Hull from "hull";
 import { Cache } from "hull/lib/infra";
 import express from "express";
+import dotenv from "dotenv";
```

Comment:
@SMK1085 it for loading `.env` vars on local instance?

Reply:
Yes, it's just way to slow to switch between multiple connectors, if you have to do it every time manually...